instrumentation/jaxrs/jaxrs-2.0/jaxrs-2.0-wildfly-testing/build.gradle.kts

```diff
@@ -21,7 +21,10 @@ dependencies {
 tasks {
   // extract wildfly dist, path is used from arquillian.xml
   val setupServer by registering(Copy::class) {
-    from(zipTree(testServer.singleFile))
+    inputs.files(testServer)
+    from({
+      zipTree(testServer.singleFile)
```

Comment:
What do these braces do?

Reply:
I think it defers the resolution of the `testServer` configuration

this is the warning message that occurs without it:

> Resolution of the configuration :instrumentation:jaxrs:jaxrs-2.0:jaxrs-2.0-wildfly-testing:testServer was attempted from a context different than the project context. Have a look at the documentation to understand why this is a problem and how it can be resolved. This behaviour has been deprecated and is scheduled to be removed in Gradle 8.0. See https://docs.gradle.org/7.4.2/userguide/viewing_debugging_dependencies.html#sub:resolving-unsafe-configuration-resolution-errors for more details.